src/main.jsx

```diff
@@ -81,6 +82,19 @@ const router = createBrowserRouter([
               },
             ],
           },
+          {
+            path: "users",
```

Comment:
minor: I use `accounts` in my PR..which do you prefer

Reply:
we can go with accounts